src/engraving/dom/tempotext.cpp

```diff
@@ -256,6 +256,11 @@ void TempoText::updateTempo()
     String s = plainText();
     s.replace(u",", u".");
     s.replace(u"<sym>space</sym>", u" ");
+    s.replace(u"≒", u"=");
+    s.replace(u"≈", u"=");
+    s.replace(u"~", u"=");
+    s.replace(u"ca.", u"=");
+    s.replace(u"approx.", u"=");
```

Comment:
Wouldn't it make more sense to replace these with `""`?

For example, if you have `\uECA5 = ca. 110`, and replace ca. with =, you get `\uECA5 = = 110` which doesn't match the regex.

Reply:
Unless it is  `♩ ca. 188`. But yes, that seems unlikely, I'll chane as per your proposal